front/lib/actions/mcp_internal_actions/constants.ts

```diff
@@ -420,6 +421,16 @@ export const INTERNAL_MCP_SERVERS: Record<
     // when the advanced_search mode is enabled.
     isRestricted: () => true,
   },
+  agent_management: {
+    id: 1011,
+    availability: "auto",
```

Comment:
Should it be `auto_hidden_builder`? Do we want users to use this tool or is it only for our global agent?

Reply:
(if `auto_hidden_builder` we can add it to a global agent but it's hidden in the agent builder)

Reply:
I realize I'm not clear on this. Asking for a quick go/meet